runtime/doc/api.txt

```diff
@@ -3195,6 +3212,11 @@ nvim_open_win({buffer}, {enter}, {*config})                  *nvim_open_win()*
                   • fixed: If true when anchor is NW or SW, the float window
                     would be kept fixed even if the window would be truncated.
                   • hide: If true the floating window will be hidden.
+                  • vertical: If creating a normal (split) window, split
+                    vertically.
+                  • belowright: If creating a normal window, split to the
```

Comment:
I thought about that, definitely could do that if that's preferred. The reason why I ended up with two properties was so that omitting one is possible (and to mirror `splitright`/`splitbelow` options). I somewhat prefer the current implementation, but I don't have strong feelings about it and I definitely understand the appeal of minimizing the number of new properties. Happy to make the change if you want :)

Reply:
I don't think `split` would eliminate the need for `vertical`. Rather it avoids the need for other, new options in the future. But I haven't thought through all the permutations so I am not certain if it's necessary. 

Currently my reasoning is:

- As a user, a `split` enum option as described above seems potentially more intuitive than a single `belowright` boolean.
- Matching the existing `nvim_parse_cmd` interface seems like a safe choice.

Reply:
> Wouldn't `topleft` be the same as `vertical=true, belowright=false`, `botright` => `vertical=true, belowright=true`, etc.?

Presumably the same reason that `:vertical` command exists :) Can always drop the `vertical` param for now and add it later if needed.

Reply:
Ah, so `vertical=true` would essentially be the same as `topleft`, and `vertical=false` would be equivalent to `botright`?

Reply:
Working on this now, and I'm realizing that the behavior of the options that match `parse_cmd` is not the same as my current implementation. The current implementation allows splitting any direction from any window, but `:belowright` / `:topleft` always create top-level splits. Imo the current functionality of this PR is more predictable and flexible than this would be. Should I use the names of the `split` options but keep the current functionality? 

I think a good enum option for the current implementation would be `top`/`bottom`/`left`/`right` because that more accurately describes how the window will split.

Reply:
Makes sense. Though should the "top" case in your table be named "above"? Compare the `:aboveleft` command to the `:topleft` command.

Reply:
Yes, good idea, I'll use `above` and `below`.

Reply:
`split` option is in, and I removed `belowright`.